bowtie/templates/report.html.j2

```diff
@@ -417,6 +435,53 @@
     const tooltipList = [...tooltipTriggerList].map(tooltipTriggerEl => new bootstrap.Tooltip(tooltipTriggerEl))
   </script>
 
+    <script>
+      var light = true;
+
+      var navbar = document.querySelector('.navbar');
+      var tableHeader = document.querySelector('.table-header');
+      var body = document.querySelector('body');
+      var sunIcon = document.querySelector('.bi-sun');
+      var moonIcon = document.querySelector('.bi-moon-stars-fill');
+      const toggler = document.querySelector('#theme-toggler');
+
+      toggler.addEventListener('click', ()=>{
+        body.setAttribute('data-bs-theme', light ? 'dark' : 'light');
+        light = !light;
+        if (!light){
+          navbar.classList.remove('bg-light');
+          navbar.classList.add('navbar-dark', 'bg-dark');
+
+          tableHeader.classList.remove('table-light');
+          tableHeader.classList.add('table-dark');
+
+          sunIcon.classList.add('hidden');
+          moonIcon.classList.remove('hidden');
+
+        } else {
+          navbar.classList.remove('navbar-dark', 'bg-dark');
+          navbar.classList.add('bg-light');
+
+          tableHeader.classList.remove('table-dark');
+          tableHeader.classList.add('table-light');
+
+          moonIcon.classList.add('hidden');
+          sunIcon.classList.remove('hidden');
+        }
+
+      })
+    </script>
+
+    <script
+    src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-ENjdO4Dr2bkBIFxQpeoTz1HIcje39Wm4jDKdf19U8gI4ddQ3GYNS7NTKfAdVQSZe"
+    crossorigin="anonymous"
+    ></script>
+   <script>
+      const tooltipTriggerList = document.querySelectorAll('[data-bs-toggle="tooltip"]')
```

Comment:
This combined with the line above redeclaring it breaks collapsing the accordion items I think.

Try running some test run (e.g. `bowtie suite -i python-jsonschema $DEVELOPMENT/JSON-Schema-Test-Suite/tests/draft7/ | bowtie report`) and then expand one of the tests and try to collapse it, it won't collapse again for me and I see:

```
Uncaught SyntaxError: redeclaration of const tooltipTriggerList
    <anonymous> file:///Users/julian/Development/bowtie/bowtie-report.html:34509
```

in the console.

Reply:
accordian collapse issue resolved